### PR TITLE
fix: remove unused authorities and refactor constants into enums

### PR DESCRIPTION
**Pull request by netroms**
## Summary

Removes some unused authorities and moves all authority string constants out of the DefaultAdminUserPopulator and into Authorities as enums.

This PR also includes: [DHIS2-15735](https://dhis2.atlassian.net/browse/DHIS2-15735)

JIRA: [DHIS2-12835](https://dhis2.atlassian.net/browse/DHIS2-12835)

[DHIS2-15735]: https://dhis2.atlassian.net/browse/DHIS2-15735?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[DHIS2-12835]: https://dhis2.atlassian.net/browse/DHIS2-12835?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: dhis-2/dhis-services/dhis-service-core/src/main/resources/i18n_global.properties [hunk 1]
```diff
@@ -216,8 +216,8 @@ F_USERGROUP_PUBLIC_ADD=Add/Update Public User Group
 F_USERGROUP_PRIVATE_ADD=Add/Update Private User Group
 F_USERGROUP_DELETE=Delete User Group
 F_USERGROUP_LIST=List User Groups
-F_USERGROUP_MANAGING_RELATIONSHIPS_ADD=Add/Update User Group Managing Relationships
-F_USERGROUP_MANAGING_RELATIONSHIPS_VIEW=View User Group Managing Relationships
+=Add/Update User Group Managing Relationships
```
Comment (jbee) [inline dhis-2/dhis-services/dhis-service-core/src/main/resources/i18n_global.properties:219]: Shouldn't the entire line(s) be deleted?

File: dhis-2/dhis-services/dhis-service-dxf2/src/test/resources/dxf2/favorites/metadata_with_visualization.json [hunk 2]
```diff
@@ -59,12 +59,12 @@
         "F_VIEW_UNAPPROVED_DATA",
         "F_INDICATOR_PUBLIC_ADD",
         "F_INDICATORGROUP_PUBLIC_ADD",
-        "F_USERGROUP_MANAGING_RELATIONSHIPS_VIEW",
+        "",
```
Comment (jbee) [inline dhis-2/dhis-services/dhis-service-dxf2/src/test/resources/dxf2/favorites/metadata_with_visualization.json:62]: This looks wrong to include empty strings. I assume we want to remove the entire string from the list

**Review by david-mackessy — APPROVED** [review]

**Comment by amcgee** [comment]
@netroms is the title of this PR correct?  It looks like the refactor to an enum isn't completed yet

**Comment by amcgee** [comment]
Oops sorry, was only looking at a single commit not the whole diff.  Ignore last comment.

**Pull request merged**